### DISCUSSION TITLE
Not able to update existing channel data using Update channel API.

### DISCUSSION THREAD

**Discussion by ramkumar-ns-dev** [Sunbird-coKreat/Community#76] [Bugs]
While trying to update new primary categories in channel API of dock-staging we are getting **404 : Resource Not Found**. But we are  able to update the newly created channels in dock-staging.

URL : `https://dockstaging.sunbirded.org/api/channel/v1/update/01345815127107174426` 

@vaivk369 Can you help on this ?

CC: @kiranharidas187

**Comment by mmanimegalai** [comment]
tagging @arun1205 cc @vinukumar-vs

**Reply by arun1205** [reply]
@ramkumar-pacewisdom cokreat does not have channel, if the data needs to be updated then it should be updated to corresponding tenant in sunbird staging only

**Reply by kiranharidas187** [reply]
@arun1205 Which means the current URL mentioned in the above thread is not correct, instead, we should use the URL: https://dev.sunbirded.org/api/channel/v1/update/01345815127107174426. right?

**Reply by arun1205** [reply]
yes
